<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PMP_PLayer</class>
 <widget class="QFrame" name="PMP_PLayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>541</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Valume</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="ValumeSlider">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="BackpushButton_5">
      <property name="text">
       <string>&lt;&lt;&lt;--</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="PausepushButton_6">
      <property name="text">
       <string>|&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="NextpushButton_8">
      <property name="text">
       <string>--&gt;&gt;&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="PlaylistWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>531</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="FilepushButton">
      <property name="text">
       <string>File</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="OptionspushButton_2">
      <property name="text">
       <string>Options</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="SearchpushButton_3">
      <property name="text">
       <string>Serch</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ExitpushButton_4">
      <property name="text">
       <string>exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>531</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item>
     <widget class="QSlider" name="horizontalSlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTimeEdit" name="timeEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
